src/app/features/tasks/components/task-create/task-create.component.scss

```diff
@@ -0,0 +1,8 @@
+:host {
+  padding: 10px;
+  margin: 10px 0;
+  border: 2px solid greenyellow;
```

Comment:
use scss variables for color, import them from separated file

Reply:
It was only for tests it's not destination styles